blackjack.py

```diff
@@ -2,137 +2,239 @@
+        else:
+            self.pcount += self.pcard[-1][0]
 
     def stand(self):
-        if self.pcount > dealer.dcount:
+        self.dhit()
+        self.dcheck()
+        if self.pcount > self.dcount:
             print('You won!')
-        elif self.pcount == dealer.dcount:
+            self.bank += self.bet
+        elif self.pcount == self.dcount:
             print('You tied!')
         else:
             print('You lost!')
-        restart()
-    
+            self.bank -= self.bet
+            self.bankrupt()
+        self.restart()
     def split(self):
         self.pcard1 = [self.pcard[0]]
         self.pcard2 = [self.pcard[1]]
-        self.pcard1.append(drawcards())
-        self.pcard2.append(drawcards())
+        self.pcard1.append(self.drawcards())
+        self.pcard2.append(self.drawcards())
         self.pcount1 = self.pcard1[0][0] + self.pcard1[1][0]
         self.pcount2 = self.pcard2[0][0] + self.pcard2[1][0]
 
-    def check(self):
-        if self.pcount > 21:
-            print('You went over 21 and lost!')
-            restart()
-
-def start():
-    print('Welcome to BlackJack')
-    name = input("What's your name? ")
-    print('Hello ' + name)
+    def setpcount(self):
+        if isinstance(self.pcard[0][0], list) == True and isinstance(self.pcard[1][0], list) == True:
+            self.pcount = self.pcard[0][0][1] + self.pcard[1][0][1]
+        elif isinstance(self.pcard[0][0], list) == True and isinstance(self.pcard[1][0], list) == False:
+            self.pcount = self.pcard[0][0][1] + self.pcard[1][0]
+        elif isinstance(self.pcard[0][0], list) == False and isinstance(self.pcard[1][0], list) == True:
+            self.pcount = self.pcard[0][0] + self.pcard[1][0][1]
+        else:
+            self.pcount = self.pcard[0][0] + self.pcard[1][0]
+            
+    def checkA(self):
+        for i in range(2):
+            if isinstance(self.pcard[i][0], list) == True:
+                while True:
+                    if self.pcard[i][0][0] == 'A':
+                        ainput = input('1 or 11? ')
+                        if ainput == '1':
+                            self.pcard[i][0][1] = 1
+                            print('\n')
+                            print(self.pcard)
+                            self.counter += 1
+                            break
+                        elif ainput == '11':
+                            self.pcard[i][0][1] = 11
+                            print('\n')
+                            print(self.pcard)
+                            self.counter += 1
+                            break
+                        else:
+                            print('Please enter 1 or 11')
+                    else:
+                        break
 
-start()
-dealer = Dealer()
-player = Player()
+    def bankrupt(self):
+        if self.bank == 0:
+            print('You went bankrupt!')
+            self.bank = 1000
+                
+game = GameManager()

```

Comment:
Get into the habit of putting these statements under `if __name__ == '__main__':` as we're going to break this file into different modules/files.

More reading here: https://stackoverflow.com/questions/419163/what-does-if-name-main-do